/**:
  ros__parameters:
    # Temporizadores:
    update_rate: 10.0
    # Trayectoria:
    trajectory_t: [0.0, 15.0, 25.0, 35.0, 45.0, 55.0, 65.0]
    trajectory_x: [0.0, 2.0, 20.0, -2.0, -2.0, 15.0, -7.0]
    trajectory_y: [0.0, -12.0, -12.0, -12.0, 4.0, 4.0, 4.0]
    trajectory_z: [7.0, 7.0, 7.0, 7.0, 7.0, 7.0, 7.0]
    # Restricciones de velocidad:
    horizontal_vel: 3.0
    vertical_vel: 2.0
    heading_vel: 1.0
    # Otros par√°metros:
    takeoff_altitude: 5.0